src/boxes.js

```diff
@@ -1,31 +1,35 @@
 SVG.Box = SVG.invent({
-  create: function(x, y, width, height) {
-    if (typeof x == 'object' && !(x instanceof SVG.Element)) {
-      // chromes getBoundingClientRect has no x and y property
-      return SVG.Box.call(this, x.left != null ? x.left : x.x , x.top != null ? x.top : x.y, x.width, x.height)
-    } else if (arguments.length == 4) {
-      this.x = x
-      this.y = y
-      this.width = width
-      this.height = height
-
-    }
+  create: function(source) {
+    var base = [0,0,0,0]
+    source = typeof source === 'string' ?
+        source.match(SVG.regex.viewbox).map(parseFloat) :
+      Array.isArray(source) ?
+        source :
+      typeof source == 'object' ?
+        [source.left != null ? source.left : source.x, source.top != null ? source.top : source.y, source.width, source.height] :
```

Comment:
What is the code coverage here? Do we actually test all of the different constructor signatures? If yes, then GREAT!